app/src/main/res/values/strings.xml

```diff
@@ -28,36 +28,34 @@ If you decide to keep it disabled, consider enabling \"%2$s\" option to improve
     </string>
     <string name="play_services_no_location_permission">
-"Play services needs Location permission to be set to \"%s\" and \"Use precise location\"."
+"Play services needs the Location permission set to “%s”, and “Use precise location” needs to be enabled."
     </string>
     <string name="play_services_no_ble_location">
-"Location estimation by Bluetooth scanning requires \"Nearby devices\" permission."
+"Location assisted by Bluetooth scanning requires the “Nearby devices” permission."
     </string>
     <string name="always_on_scanning_settings">Always-on scanning settings</string>
-    <string name="always_on_wifi_scanning_enabled_but_disallowed">Always-on Wi-Fi scanning is enabled, but not allowed (no Location permission).</string>
+    <string name="always_on_wifi_scanning_enabled_but_disallowed">Always-on Wi-Fi scanning is enabled, but the Location permission isn’t allowed.</string>
     <string name="always_on_wifi_scanning_allowed">Always-on Wi-Fi scanning is allowed.</string>
     <string name="always_on_wifi_scanning_not_allowed">Always-on Wi-Fi scanning is disabled.</string>
     <string name="always_on_bluetooth_scanning_allowed">Always-on Bluetooth scanning is allowed.</string>
-    <string name="always_on_bluetooth_scanning_enabled_but_disallowed">Always-on Bluetooth scanning is enabled, but not allowed (no \"Nearby devices\" permission).</string>
+    <string name="always_on_bluetooth_scanning_enabled_but_disallowed">Always-on Bluetooth scanning is enabled, but the “Nearby devices” permission isn’t allowed.</string>
     <string name="always_on_bluetooth_scanning_disabled">Always-on Bluetooth scanning is disabled.</string>
 
     <string name="play_store_obb_permission">
-"To install apps that include OBB files (usually the ones that are over 100 MB in size), Play Store needs to have \"Files and media\" permission.
-
-After this permission is granted, Play Store needs to be fully restarted with \"Force stop\" button."
+"To install apps with OBB files (such as games), allow the “Files and media” for Play Store and restart it with “Force stop”."
     </string>
     <string name="no_play_games">
-"A lot of games won\'t function properly without Play Games app (they will show misleading \"Missing permissions\" prompt)."
+"Many games need the Play Games app, or they’ll show a misleading “Missing permissions” prompt."
     </string>
     <string name="install_play_games">Install Play Games</string>
 
     <string name="push_notifications">Push notifications</string>
     <string name="play_services_no_battery_exemption">
-"When device is idle, push notifications will be delayed to increase battery life.
-To receive them promptly, set \"Battery usage\" to \"Unrestricted\" in Play services settings."
+"When device is idle, push notifications will be delayed to improve battery life.
+To receive them immediately, set “Battery usage” to “Unrestricted” in Play services app info."
```

Comment:
Disagree with the use of "immediately" here. Setting “Battery usage” to “Unrestricted” won't make them immediate.